### ISSUE TITLE
Provide aggregating helper for access

### ISSUE THREAD

**Issue by tecimovic** [project-chip/zap#347]
(This is following a discussion on a zoom call, recording requirements so we don't forget...)

1.) Add a new helper `accessAggregate` that will NOT iterate over all access elements, but provide an aggregation of all multiple elements for easier usage, since in most case, all you need is highest or lowest role that is allowed for a given operation.

2.) If certain operation is not mentioned at all, provide a constant `NONE` or something like that.

Idea is that instead of using {{access}}, which quickly becomes iterative nightmare, since each entity can have unlimited number of access triplets attached to it, we will do this:

```
{{#accessAggregate type="attribute" includeDefault="false"}}
   {{readHighestPrivilege}}  <- shows you the lowest privilege that is allowed to read.
   {{readLowestPrivilege}}   <- shows you the highest privilege that is allowed to read.

  {{#if readHighestPrivilege == NONE}}
      ...
  {{/if}}
etc....

{{/accessAggregate}}
```

So basically it avoids having to deal with iteration.

**Comment by tecimovic** [comment]
Final implementation (see access.zapt for example):
{{#access_aggregate entity="attribuye" includeDefault="false/true"}}
  You can use following here:
     {{readLowest}} / {{readHighest}}
     {{writeLowest}} / {{writeHighest}}
     {{invokeLowest}} / {{invokeHighest}}
     {{fabric-sensitive}}
     {{fabric-scoped}}
{{/access_aggregate}}

In short the helper will populate the context with:
   every "modified" will be either true or false, depending whether it was present in the access array
   every "operation" will be present as operationHighest and operationLowest, where value will be the highest and lowest role listed in the access list for a given operation.
